### PR TITLE
API to check if glusterd2 service is up

### PR DESCRIPTION
**Pull request by Madhu-1**
if glusterd2 consumers  (ex:- HEKETI )wanted to check the status of glusterd2 service.
they can consume this URL to check the status.

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@aravindavk  @prashanthpai  please review

**Comment by aravindavk** [comment]
Change URL to `/ping` and respond accordingly. I agree with @prashanthpai, it is not providing the health of Glusterd/cluster

**Comment by Madhu-1** [comment]
@prashanthpai  this is required to check the status fo glusterd2 service, not the cluster status.
in heketi, we use to check the glusterd status by ```systemctl status glusterd.service```
so that this URL will be consumed to check the status of service.

and this URL will be used in the kubernetes world to check liveness of service.

**Comment by Madhu-1** [comment]
@aravindavk  changed URL to ```/ping```

File: glusterd2/servers/rest/rest.go [hunk 1]
```diff
@@ -135,3 +135,11 @@ func (r *GDRest) listEndpointsHandler() http.HandlerFunc {
 		restutils.SendHTTPResponse(ctx, w, http.StatusOK, resp)
 	})
 }
+
+//Ping URL for glusterd2
```
Comment (prashanthpai) [inline glusterd2/servers/rest/rest.go]: This need not be exported.

**Pull request merged**